pavelib/quality.py

```diff
@@ -1,203 +1,97 @@
 """
-Check code quality using pep8, pylint, and diff_quality.
+Check code quality using PEP8, Pylint, and diff_quality.
 """
-from paver.easy import sh, task, cmdopts, needs, BuildFailure
+from __future__ import print_function
 import os
-import re
 
-from .utils.envs import Env
+from paver.easy import task, cmdopts, needs, BuildFailure
 
-ALL_SYSTEMS = 'lms,cms,common'
+from pavelib.pep8 import pep8
+from pavelib.pylint import get_clean_report_directory
+from pavelib.pylint import get_pylint_reports
+from pavelib.pylint import pylint
+from pavelib.utils.cmd import shell
+from pavelib.utils.envs import Env
+
+ALL_SYSTEMS = [
+    'lms',
+    'cms',
+    'common',
+    'docs',
+    'scripts',
+    'pavelib',
+    'openedx',
+]
 
 
 @task
 @needs('pavelib.prereqs.install_python_prereqs')
 @cmdopts([
-    ("system=", "s", "System to act on"),
+    ('system=', 's', 'System to act on'),
 ])
 def find_fixme(options):
     """
-    Run pylint on system code, only looking for fixme items.
+    Run Pylint on system code, only looking for FIXME items
     """
-    num_fixme = 0
-    systems = getattr(options, 'system', ALL_SYSTEMS).split(',')
-
-    for system in systems:
-        # Directory to put the pylint report in.
-        # This makes the folder if it doesn't already exist.
-        report_dir = (Env.REPORT_DIR / system).makedirs_p()
-
-        apps = [system]
-
-        for directory in ['djangoapps', 'lib']:
-            dirs = os.listdir(os.path.join(system, directory))
-            apps.extend([d for d in dirs if os.path.isdir(os.path.join(system, directory, d))])
-
-        apps_list = ' '.join(apps)
-
-        pythonpath_prefix = (
-            "PYTHONPATH={system}:{system}/lib"
```

Comment:
This line was missing a colon at the end, which resulted in a path like `lms/libcommon/djangoapps`.